src/function-builder.ts

```diff
@@ -127,28 +164,38 @@ export class FunctionBuilder {
    * functions.region('us-east1', 'us-central1')
    */
   region(...regions: Array<typeof SUPPORTED_REGIONS[number]>): FunctionBuilder {
-    if (assertRegionsAreValid(regions)) {
-      this.options.regions = regions;
-      return this;
-    }
+    assertRegionsValidity(regions);
+
+    this.options.regions = regions;
+
+    return this;
   }
 
   /**
    * Configure runtime options for the function.
    * @param runtimeOptions Object with three optional fields:
-   * 1. memory: amount of memory to allocate to the function, possible values
+   * 1. failurePolicy: failure policy policy of the function, boolean `true` is
+   *    equivalent to providing an empty policy.
```

Comment:
This phrasing is a bit awkward here. I think something like this would sound better: `failurePolicy: failure policy of the function, with boolean true being equivalent to providing an empty retry object`. Last part could also be changed to "...to providing a default failure policy".

Reply:
3a7c8512a60996b0848472b467d5d2a80f8e3ab9